.clang-format

```
@@ -85,4 +82,11 @@ SpacesInSquareBrackets: false
 Standard:        Cpp11
 TabWidth:        8
 UseTab:          Never
-...
+---
+Language: ObjC
+ColumnLimit: 120
+AlignAfterOpenBracket: Align
+ObjCBlockIndentWidth: 2
+ObjCSpaceAfterProperty: false
+ObjCSpaceBeforeProtocolList: false
+...
```

I think / hope this should resolve both CI failures from https://github.com/pytorch/vision/actions/runs/5312942594/jobs/9618529905?pr=7677

```suggestion
...

```